beacon-chain/blockchain/metrics.go

```diff
@@ -100,13 +96,6 @@ var (
 		Name: "beacon_reorg_total",
 		Help: "Count the number of times beacon chain has a reorg",
 	})
-	sentBlockPropagationHistogram = promauto.NewHistogram(
```

Comment:
This is a pretty useful metric, can we continue using it ? 
We just need to hook `captureSentTimeMetric` for blocks received through gossip

Reply:
What you are looking for is `block_arrival_latency_milliseconds`

It's hooked up and used today: https://github.com/prysmaticlabs/prysm/blob/master/beacon-chain/sync/validate_beacon_blocks.go#L100